cogs/modmail.py

```diff
@@ -0,0 +1,394 @@
+                em = discord.Embed(color=discord.Color.green())
+                em.set_author(name='Previous Logs', icon_url=user.avatar_url)
+                embeds.append(em)
+
+            date = dateutil.parser.parse(entry['created_at'])
+            new_day = date.strftime(r'%d %b %Y')
+            
+            key = entry['key']
+            user_id = entry['user_id']
+            log_url = f"https://logs.modmail.tk/{user_id}/{key}"
+            
+            if not entry['open']:  # only list closed threads
+                fmt += f"[`{key}`]({log_url})\n"
+
+                if current_day != new_day or index == len(logs) - 2:
+                    embeds[-1].add_field(name=current_day, value=fmt)
+                    current_day = new_day
+                    fmt = ''
+        
+        session = PaginatorSession(ctx, *embeds)
+        await session.run()
+
+    @commands.command()
+    @trigger_typing
+    async def reply(self, ctx, *, msg=''):
+        '''Reply to users using this command.
+        
+        Supports attachments and images as well as automatically embedding image_urls.
+        '''
+        ctx.message.content = msg
+        thread = await self.bot.threads.find(channel=ctx.channel)
+        if thread:
+            await thread.reply(ctx.message)
+    
+    @commands.command()
+    async def edit(self, ctx, message_id: Optional[int]=None, *, new_message):
+        '''Edit a message that was sent using the reply command.
+        
+        If no message_id is provided, that last message sent by a mod will be edited.
+
+        `[message_id]` the id of the message that you want to edit.
+        `<new_message>` is the new message that will be edited in.
+        '''
+        thread = await self.bot.threads.find(channel=ctx.channel)
+
+        if thread is None:
+            return 
+
+        linked_message_id = None
+            
+        async for msg in ctx.channel.history():
+            if message_id is None and msg.embeds:
+                em = msg.embeds[0]
+                if 'Moderator' not in str(em.footer.text):
+                    continue
+                linked_message_id = int(re.findall(r'\d+', em.author.url)[0])
+                break 
+            elif message_id and msg.id == message_id:
+                url = msg.embeds[0].author.url 
+                linked_message_id = int(re.findall(r'\d+', url)[0])
+                break
+        
+        if not linked_message_id:
+            raise commands.UserInputError
+
+        await thread.edit_message(linked_message_id, new_message)
+        await ctx.message.add_reaction('✅')
+
+    @commands.command()
+    @trigger_typing
+    @commands.has_permissions(manage_channels=True)
+    async def contact(self, ctx, *, user: discord.Member):
+        '''Create a thread with a specified member.'''
+
+        exists = await self.bot.threads.find(recipient=user)
+        if exists:
+            return await ctx.send('Thread already exists.')
```

Comment:
Maybe mention the exist channel?

```suggestion
            return await ctx.send(f'Thread already exists: {exists.mention}.')
```